### PR TITLE
fix download on thankyou screen

### PR DESCRIPTION
**Pull request by tejassonar**
Fixes fix download option on thankyou screen

### CODE REVIEW

File: src/Donations/Micros/PaymentStatus/SuccessfulDonation.tsx [hunk 1]
```diff
@@ -45,7 +43,11 @@ function SuccessfulDonation({ donation, sendToReturn }: any) {
         projectDetails={projectDetails}
         donation={donation}
       />
-      <ImageComponent projectDetails={projectDetails} donation={donation} />
+      <ImageComponent
+        projectDetails={projectDetails}
+        donation={donation}
+        imageRef={imageRef}
```
Comment (norbertschuler) [inline src/Donations/Micros/PaymentStatus/SuccessfulDonation.tsx:49]: Too much code reformatting to easily review this patch 🥲 Is this the only real change?

**Review by norbertschuler — APPROVED** [review]
I was not aware the image download was broken (but just tested it on paydev.pp.eco). I can confirm it working again in this PR.

**Comment by norbertschuler** [comment]
Btw. nothing happens if I try to download that image on my Chrome browser on my iPhone, but Safari support the download on iOS. So this has no priority.

File: src/Donations/Micros/PaymentStatus/SuccessfulDonation.tsx (hunk 1 above)
Comment (tejassonar) [reply src/Donations/Micros/PaymentStatus/SuccessfulDonation.tsx:49]: Yes. The ref was not being passed, because of which the download was failing.

**Pull request merged**